packages/clone-testing/Cargo.toml

```diff
@@ -13,9 +13,9 @@ description = "Adapter for interacting with cw-multi-test via the cw-orchestrato
 cosmwasm-std = { workspace = true }
 
 # This package depends on published cw-orch version due to the clone-testing-cw-multi-test dependency
-cw-orch-core = { version = "2.0.0" }
-cw-orch-mock = { version = "0.24.0" }
-cw-orch-daemon = { version = "0.25.0" }
+cw-orch-core = { path = "../cw-orch-core" }
```

Comment:
should be able to use `workspace = true` no? 

Otherwise you'll have trouble publishing this package.

Reply:
Yep